### PR TITLE
Grant credits for filling in profile streams by user.

### PR DESCRIPTION
**Pull request by AndreyTepaykin**

### CODE REVIEW

File: platform/plugins/Assets/config/plugin.json [hunk 1]
```diff
@@ -218,14 +218,30 @@
...
 					"Streams/user/lastName": 1,
 					"Streams/user/icon": 10,
 					"Places/user/location": 10,
 					"Streams/greeting/": 10
+				},
+				"forStreams": {
+					"Streams/user/birthday": 1,
+					"Streams/user/icon": 5,
+					"Streams/user/gender": 1,
+					"Streams/user/height": 1,
```
Comment (EGreg) [inline platform/plugins/Assets/config/plugin.json:237]: Add also Streams/user/firstname and Streams/user/lastname each 5

File: platform/plugins/Assets/handlers/Assets/after/Streams_save.php [hunk 2]
```diff
@@ -1,15 +1,68 @@
 <?php
 function Assets_after_Streams_save($params) {
+	nft_update_attributes_relations($params);
+	grant_credits_for_filling_personal_streams($params);
+	grant_credits_for_invited_users($params);
+}
+
+function nft_update_attributes_relations ($params) {
```
Comment (EGreg) [inline platform/plugins/Assets/handlers/Assets/after/Streams_save.php]: Rename to Assets_NFT_update... and Asserts_grant_credits...

File: platform/plugins/Assets/text/Assets/content/en.json [hunk 3]
```diff
@@ -49,11 +49,8 @@
     },
     "credits": {
         "InviteAcceptedBy": "Invite accepted by {{invitedUserName}}",
-        "Streams_user_firstname": "{{invitedUserName}} entered first name",
-        "Streams_user_lastname": "{{invitedUserName}} entered last name",
-        "Streams_user_icon": "{{invitedUserName}} entered icon",
-        "Places_user_location": "{{invitedUserName}} entered location",
-        "Streams_greeting": "{{invitedUserName}} entered greeting",
+        "IvitedUserFilledStream": "{{invitedUserName}} filled {{FilledStreamTitle}}",
```
Comment (EGreg) [inline platform/plugins/Assets/text/Assets/content/en.json]: You misspelled "Invited"!!

File: platform/plugins/Users/classes/Users.php [hunk 4]
```diff
@@ -39,13 +39,14 @@ static function isCommunityId($userId)
 	 * @method isCustomIcon
 	 * @static
 	 * @param {String} $icon
+	 * @param {Boolean} [$imported=true] - If false, don't mean imported icon as custom
 	 * @return {boolean}
 	 */
-	static function isCustomIcon ($icon) {
+	static function isCustomIcon ($icon, $imported=true) {
```
Comment (EGreg) [inline platform/plugins/Users/classes/Users.php]: Instead make it $unlessImported = false

**Review by EGreg — CHANGES_REQUESTED** [review]

File: platform/plugins/Assets/config/plugin.json (hunk 1 above)
Comment (AndreyTepaykin) [reply platform/plugins/Assets/config/plugin.json:237]: Done

File: platform/plugins/Assets/text/Assets/content/en.json (hunk 3 above)
Comment (AndreyTepaykin) [reply platform/plugins/Assets/text/Assets/content/en.json]: Fixed

File: platform/plugins/Assets/handlers/Assets/after/Streams_save.php (hunk 2 above)
Comment (AndreyTepaykin) [reply platform/plugins/Assets/handlers/Assets/after/Streams_save.php]: Done

File: platform/plugins/Users/classes/Users.php (hunk 4 above)
Comment (AndreyTepaykin) [reply platform/plugins/Users/classes/Users.php]: Done

**Review by EGreg — APPROVED** [review]

**Pull request merged**